docs/fsharp/language-reference/results.md

```diff
@@ -67,16 +67,16 @@ let validateRequest reqResult =
 let test() = 
     // Now, create a Request and pattern match on the result.
     let req1 = { Name = "Phillip"; Email = "phillip@contoso.biz" }
-    let res1 = validateRequest (OK req1)
+    let res1 = validateRequest (Ok req1)
     match res1 with
-    | Ok req -> printfn "My request was valid! Name: %s Email %s" req1.Name req1.Email
+    | Ok req -> printfn "My request was valid! Name: %s Email %s" req.Name req.Email
```

Comment:
This should stay the same - `req` isn't defined anywhere

Reply:
@cartermp there is a pattern matching, right?
`| Ok req -> ...`

Reply:
Ah, for some reason I couldn't see what was going on. I blame that on a long day.